views/components/tabbar.erb

```diff
@@ -1,15 +1,13 @@
 <div class="sm:block mb-6">
   <nav class="isolate flex divide-x divide-gray-200 rounded-lg shadow" aria-label="Tabs">
-    <% tabs.each_with_index do |(label, url), i| %>
+    <% tabs.each_with_index do |(label, url, active), i| %>
+      <% active ||= (request.path == url) %>
```

Comment:
Seems like you would only want to do this if `active` is `nil`.  If you are explicitly passing `false`, it seems unnecessary to do the path check.

```suggestion
      <% active = (request.path == url) if active.nil? %>
```